code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -110,7 +110,7 @@
 
 /datum/chemical_reaction/sorium_vortex/on_reaction(datum/reagents/holder, created_volume)
 	var/turf/simulated/T = get_turf(holder.my_atom)
-	goonchem_vortex(T, 1, 5, 6)
+	goonchem_vortex(T, 1, round(created_volume / 33), round(created_volume / 25))
```

Comment:
Example cases for balance design purposes:
To get 5 tiles of range, `created_volume = 33*5 = 165 units`
To get 6 steps of pull, `created_volume = 25*6 = 150 units`

(The same holds true for LDM)